runtime/j9vm/jvm.c

```diff
@@ -988,56 +1011,59 @@ preloadLibraries(void)
+	globalGetVMs = (GetVMs)GetProcAddress(vmDLL, (LPCSTR)GET_JAVA_VMS_ENTRYPOINT);
 	if (!globalCreateVM || !globalGetVMs) {
 		FreeLibrary(vmDLL);
-		fprintf(stderr,"jvm.dll failed to load: global entrypoints not found\n");
+		fprintf(stderr, "jvm.dll failed to load: global entrypoints not found\n");
 		return FALSE;
 	}
 	j9vm_dllHandle = vmDLL;
 
-	threadDLL = (HINSTANCE) preloadLibrary(J9_THREAD_DLL_NAME, TRUE);
-	f_threadGlobal = (ThreadGlobal) GetProcAddress (threadDLL, (LPCSTR) "omrthread_global");
-	f_threadAttachEx = (ThreadAttachEx) GetProcAddress (threadDLL, (LPCSTR) "omrthread_attach_ex");
-	f_threadDetach = (ThreadDetach) GetProcAddress (threadDLL, (LPCSTR) "omrthread_detach");
-	f_monitorEnter = (MonitorEnter) GetProcAddress (threadDLL, (LPCSTR) "omrthread_monitor_enter");
-	f_monitorExit = (MonitorExit) GetProcAddress (threadDLL, (LPCSTR) "omrthread_monitor_exit");
-	f_monitorInit = (MonitorInit) GetProcAddress (threadDLL, (LPCSTR) "omrthread_monitor_init_with_name");
-	f_monitorDestroy = (MonitorDestroy) GetProcAddress (threadDLL, (LPCSTR) "omrthread_monitor_destroy");
-	f_threadLibControl = (ThreadLibControl) GetProcAddress (threadDLL, (LPCSTR) "omrthread_lib_control");
-	f_setCategory = (SetCategory) GetProcAddress (threadDLL, (LPCSTR) "omrthread_set_category");
-	f_libEnableCPUMonitor = (LibEnableCPUMonitor) GetProcAddress (threadDLL, (LPCSTR) "omrthread_lib_enable_cpu_monitor");
-	f_threadSleep = (ThreadSleep) GetProcAddress (threadDLL, (LPCSTR) "omrthread_sleep");
-	if (!f_threadGlobal || !f_threadAttachEx || !f_threadDetach || !f_monitorEnter || !f_monitorExit || !f_monitorInit
-		|| !f_monitorDestroy || !f_threadLibControl || !f_setCategory || !f_libEnableCPUMonitor || !f_threadSleep
+	threadDLL = (HINSTANCE)preloadLibrary(J9_THREAD_DLL_NAME, TRUE);
+	f_threadGlobal = (ThreadGlobal)GetProcAddress(threadDLL, (LPCSTR) "omrthread_global");
+	f_threadAttachEx = (ThreadAttachEx)GetProcAddress(threadDLL, (LPCSTR) "omrthread_attach_ex");
+	f_threadDetach = (ThreadDetach)GetProcAddress(threadDLL, (LPCSTR) "omrthread_detach");
+	f_monitorEnter = (MonitorEnter)GetProcAddress(threadDLL, (LPCSTR) "omrthread_monitor_enter");
+	f_monitorExit = (MonitorExit)GetProcAddress(threadDLL, (LPCSTR) "omrthread_monitor_exit");
+	f_monitorInit = (MonitorInit)GetProcAddress(threadDLL, (LPCSTR) "omrthread_monitor_init_with_name");
+	f_monitorDestroy = (MonitorDestroy)GetProcAddress(threadDLL, (LPCSTR) "omrthread_monitor_destroy");
+	f_threadLibControl = (ThreadLibControl)GetProcAddress(threadDLL, (LPCSTR) "omrthread_lib_control");
+	f_setCategory = (SetCategory)GetProcAddress(threadDLL, (LPCSTR) "omrthread_set_category");
+	f_libEnableCPUMonitor = (LibEnableCPUMonitor)GetProcAddress(threadDLL, (LPCSTR) "omrthread_lib_enable_cpu_monitor");
+	f_threadSleep = (ThreadSleep)GetProcAddress(threadDLL, (LPCSTR) "omrthread_sleep");
+	if (
+		!f_threadGlobal || !f_threadAttachEx || !f_threadDetach
```

Comment:
Please replace `!` with `NULL ==` (and parentheses) like you did elsewhere in this change.